code/src/components/ProductInfo.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { useSelector } from "react-redux";
+// import { NotFound } from 'components/NotFound'
+import styled from "styled-components/macro";
+
+export const ProductInfo = () => {
+  const item = useSelector((store) => store.productsList.products);
+
+  if (!item) return null;
+
+  return (
+    <div>
+      {item.product && <ProductName>{item.product.product_name}</ProductName>}
+      {item.product && <Brand>{item.product.brands}</Brand>}
+      <Container>
+        {item.product && (
+          <NutrimentsText>
+            Fat: {item.product.nutriments.fat_100g}
+          </NutrimentsText>
+        )}
+        {item.product && (
+          <NutrimentsText>
+            Calories: {item.product.nutriments.energy_100g}
+          </NutrimentsText>
+        )}
+        {item.product && (
+          <NutrimentsText>
+            Sugars: {item.product.nutriments.sugars_100g}
+          </NutrimentsText>
+        )}
+        {item.product && (
+          <NutrimentsText>
+            Proteins: {item.product.nutriments.proteins_100g}
+          </NutrimentsText>
+        )}
+        {item.product && <NutrimentsGrams>per 100 g</NutrimentsGrams>}
+      </Container>
+      {item.status === 0 && (
+        <ProductNotFound>
+          Oh no! {item.status_verbose}{" "}
+          <span role="img" aria-label="angry-beast">
+            👹
+          </span>
+        </ProductNotFound>
+      )}
```

Comment:
Awesome! Since you now have an input field instead of a scanner, you give the user the possibility to write any characters (and not just numbers). The app crashes if you write letters. Can you make a check for this as well in some way or make sure the user only can type numbers? 😎